spring-boot3-dummy/src/main/java/org/springframework/boot/autoconfigure/service/connection/ConnectionDetails.java

```diff
@@ -0,0 +1,8 @@
+package org.springframework.boot.autoconfigure.service.connection;
```

Comment:
I'm missing the copyright notice, which I assume isn't even going to be our default copyright notice, right?